src/ui/buttons/styles/custom.js

```diff
@@ -10,11 +10,19 @@ export const customStyle = ({ custom } : {| custom? : CustomStyle |}) : string =
 
     const { css } = custom || {};
 
+    let heightStyle = '';
     let style = Object.keys(css).reduce((acc, key) => {
+        if (key === 'height') {
```

Comment:
I'm not sure the exact custom styling requirements, but there may be some overlap with what's in SPB for the new card fields here: https://github.com/paypal/paypal-smart-payment-buttons/blob/ee31f1d4ee9bea646dcb21b5b7cf88fe4f8d974c/src/card/lib/card-utils.test.js#L372. Just wanted to mention it incase there's some logic we can copy/paste.

Reply:
Actually will be having the merchant enter in custom styling as they would in CSS with no conversion.